server/src/inmantals/lsp_types.py

```diff
@@ -53,6 +53,18 @@ class DiagnosticSeverity(Enum):
     Hint: int = 4
 
 
+class MessageType(Enum):
+    """
+    Message type.
+    `LSP spec 3.15 <https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#window_showMessage>`__
```

Comment:
```suggestion
```
I've moved this to a general spec reference at the top of the document (in the current master).